yaml/src/main/java/com/fasterxml/jackson/dataformat/yaml/YAMLParser.java

```diff
@@ -415,12 +419,22 @@ protected JsonLocation _locationFor(Mark m)
 
     // Note: SHOULD override 'getTokenLineNr', 'getTokenColumnNr', but those are final in 2.0
 
+    /**
+     * Since the parserImpl cannot be replaced allow subclasses to at least be able to
+     * influence the events being consumed.
+     *
+     * A particular use case is working around the lack of anchor and alias support to
+     * emit additional events.
+     */
+    protected Event getEvent() {
```

Comment:
This should probably be also called from somewhere? Most likely from

````
    public JsonToken nextToken() throws IOException  {

    }
````

will make change as well.

Reply:
Good catch.